Server/Server.Core/Server.Core.Static/Server.cpp

```diff
@@ -664,10 +630,10 @@ void Server::directMessageCreateRequest(const std::shared_ptr<Connection>& clien
 }
 void Server::defaultRequest() const
 { 
-    Base::Logger::FileLogger::getInstance().log
+    FileLogger::getInstance().log
     (
         "Unknown command received", 
-        Base::Logger::LogLevel::ERR
+        LogLevel::ERR
     );
 }
-}  /// namespace Server
\ No newline at end of file
+}  // namespace Server
```

Comment:
Fix end line, please

Reply:
In all files

Reply:
I did it. In our files we have 
![image](https://user-images.githubusercontent.com/70116007/169515886-3f751755-e04b-481c-ae55-2a9294c1665f.png)
My decision as the same as we have:
![image](https://user-images.githubusercontent.com/70116007/169516091-d1a8524e-ff7a-48e2-814f-2451203c17ab.png)
Or what you mean?


> In all files

Reply:
Hmmm. What do I mean by that....
![image](https://user-images.githubusercontent.com/4313547/169522742-9d52a226-142c-4085-a4c2-62d2cdecb776.png)